scripts/shellcheck.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+set -o pipefail
+
+shellcheck --version
+# lint all *.sh files
+find . -type f -name '*.sh' ! -path './node_modules/*' -print0 | xargs -0 shellcheck
```

Comment:
perhaps we should ignore `.yarn/` path as well

Reply:
Probably yes, lest add it there.